compare_run.py

```diff
@@ -133,7 +133,7 @@ def _run_spec(version, spec, result_hosts, env, settings, tmpdir):
             log=log,
             result_hosts=result_hosts,
             sample_mode='reservoir',
-            action='queries'
+            action=['queries', 'load_data']
```

Comment:
if a spec doesn't have `load_data` section, would it be fine?

Reply:
yes, it will ignore missing actions and run the ones that are present